lib/pengine/utils.c

```diff
@@ -69,8 +69,14 @@ pe_can_fence(const pcmk_scheduler_t *scheduler, const pcmk_node_t *node)
         /* Remote nodes are marked online when we assign their resource to a
          * node, not when they are actually started (see remote_connection_assigned)
          * so the above test by itself isn't good enough.
+         *
+         * This behavior is now the default.  The user can opt out of it (and
```

Comment:
It's not clear what "This behavior" refers to, unless my tired eyes are misreading. It sounds like we're saying "Remote nodes are arked online when..." is now the default behavior. But we're actually saying the current default is that remote nodes can't be fenced by an inquorate node.